src/plugins/stt/azure/AzureSpeechRecognition.ts

```diff
@@ -10,10 +11,15 @@ import { AzureToken } from './AzureToken';
 export class AzureSpeechRecognition {
   recognizer: SpeechRecognizer | undefined;
 
+  isRunning(): boolean {
+    return this.recognizer !== undefined;
+  }
+
   async start(
     subscriptionKey: string,
     region: string,
-    onSpeechRecognized: (transcript: string) => void,
+    onSpeechRecognized: (transcript: string, isFinal: boolean) => void,
```

Comment:
I forgot to explain in the PR description, but these are changes that Trilogy made on top of our SDK. I tried to replicate only the ones that made some sense.

But I don't think it will be a big problem, because even though it is a breaking change, it will only be affected by those who update their SDK version.

Reply:
Of course, but it's still a breaking change, if anyone wants to update our SDK to use a new feature will have their app broken because of that. When developing an SDK every move needs to be thought throughly to minimize breaking changes. TBH the idea of having separate listeners actually sound good to me (aside from preventing the breaking change)

Reply:
We decided to continue with the breaking change as no one is using the AzureSpeechRecognition plugin at the moment.